src/funding/venmo/config.jsx

```diff
@@ -13,17 +12,8 @@ export function getVenmoConfig() : FundingSourceConfig {
     return {
         ...DEFAULT_FUNDING_CONFIG,
 
-        requires: {
-            popup:          true,
-            native:         true
```

Comment:
This `requires.native` setting is what prevents Venmo from showing on unsupported mobile user agent string. I understand we need to remove this to support Venmo Desktop. But we do need to migrate this logic somewhere else so venmo does not show on unsupported mobile browsers.